AndelaOSP/art-dashboard
# #160136598 View User Detail

## What does this PR do?
Users can be viewed in detail
## Description of Task to be completed?
- add user detail container and component
- add logic to pass props to user detail component
- add reducer, action to fetch one user
- add tests
- add logic to check if user exists. If not fetch data form the API.
## How should this be manually tested?
`npm test` OR 
- Login to ART
- Navigate to the user list.
- click on one user. You see more details about the user
## What are the relevant pivotal tracker stories?
[#160136598](https://www.pivotaltracker.com/story/show/160136598)
## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="1298" alt="screen shot 2018-09-07 at 10 40 05" src="https://user-images.githubusercontent.com/6042152/45210630-ca15ce80-b298-11e8-8e3f-49f14e476da4.png">
